CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog #
 
+## Unreleased ##
+ * Add ``DJANGO_FIDO_RP_NAME`` setting.
```

Comment:
Should be noted that this might be a breaking change if `rp_name` was previously set as class attribute.

Reply:
We discussed that it's actually not a breaking change. `rp_name` can still be overridden by setting a class attribute.